docs/minio-client-complete-guide.md

```diff
@@ -435,34 +436,36 @@ ENVIRONMENT VARIABLES:
 
 ```sh
 mc rm play/mybucket/myobject.txt
-Removed ‘play/mybucket/myobject.txt’.
+Removing `play/mybucket/myobject.txt`.
 ```
 *Example: Remove an encrypted object.*
 
 ```sh
 mc rm --encrypt-key "play/mybucket=32byteslongsecretkeymustbegiven1" play/mybucket/myobject.txt
-Removed ‘play/mybucket/myobject.txt’.
+Removing `play/mybucket/myobject.txt`.
 ```
 
 *Example: Recursively remove a bucket and all its contents. Since this is a dangerous operation, you must explicitly pass `--force` option.*
 
 ```sh
-mc rm --recursive --force play/myobject
-Removed ‘play/myobject/newfile.txt’.
-Removed 'play/myobject/otherobject.txt’.
+mc rm --recursive --force play/mybucket
+Removing `play/mybucket/newfile.txt`.
+Removing `play/mybucket/otherobject.txt`.
+Removing `play/mybucket`.
 ```
 
-*Example: Remove all incompletely uploaded files from `mybucket`.*
+*Example: Remove all uploaded incomplete files for an object.*
 
 ```sh
-mc rm  --incomplete --recursive --force play/mybucket
-Removed ‘play/mybucket/mydvd.iso’.
-Removed 'play/mybucket/backup.tgz’.
+mc rm --incomplete --recursive --force play/mybucket/myobject.1gig
+Removing `play/mybucket/myobject.1gig`.
 ```
-*Example: Remove object only if its created older than one day.*
+*Example: Remove object and output a message only if the object is created older than one day. Otherwise, the command stays quiet and nothing is printed out.*
```

Comment:
I think it makes sense if we show another example better than this, like removing a set of objects which are older than one day, `mc rm --recursive --older-than=1 play/mybucket/`

What do you think @ebozduman ?

Reply:
Sure.

Reply:
Done.